charts/airbyte/templates/pod-sweeper/deployment.yaml

```diff
@@ -40,6 +43,16 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: KUBERNETES_AUTH_TRYSERVICEACCOUNT
```

Comment:
can we add the link to the fabric client documentation here to make it clear for users what is consuming these?